packages/perseus/src/widgets/__tests__/explanation.test.ts

```diff
@@ -41,167 +85,184 @@ describe("Explanation", function () {
         // Arrange
+        const matchMediaMock = jest
+            .spyOn(window, "matchMedia")
+            .mockImplementation(
+                getMatchMediaMockFn(
+                    false,
+                    "(prefers-reduced-motion: no-preference)",
+                ),
+            );
         renderQuestion(question1);
 
-        // Act - expand with a click
-        const expandButton = screen.getByRole("button", {
-            name: "[Explanation]",
-        });
-        expandButton.focus();
-        await userEvent.keyboard(" ");
+        // Act - expand
+        await userEvent.click(
+            screen.getByRole("button", {name: "Explanation"}),
+        );
 
-        // Assert
-        // get asserts if it doesn't find a single matching element
-        expect(screen.getByText("This is an explanation")).toBeVisible();
+        // Assert - transition when revealing
+        expect(screen.getByTestId("content-container").className).not.toContain(
+            "transitionExpanded",
+        );
 
-        // Act - collapse with a click
-        const collapseButton = screen.getByRole("button", {
-            name: "[Hide explanation!]",
-        });
-        collapseButton.focus();
-        await userEvent.keyboard(" ");
+        // Act - collapse
+        await userEvent.click(
+            screen.getByRole("button", {name: "Hide explanation!"}),
+        );
 
-        // Assert
-        expect(screen.queryByText("This is an explanation")).toBeNull();
+        // Assert - transition when concealing
+        expect(screen.getByTestId("content-container").className).not.toContain(
+            "transitionCollapsed",
+        );
+
+        // Clean-up
+        matchMediaMock.mockRestore();
     });
 
-    it("can be collapsed", async function () {
+    it("communicates changes to its parent by using the provided 'onChange' callback", () => {
         // Arrange
-        renderQuestion(question1);
-
-        // Act
-        const expandLink = screen.getByRole("button", {expanded: false});
-        await userEvent.click(expandLink); // expand and then
-        const collapseLink = screen.getByRole("button", {
-            expanded: true,
+        // @ts-expect-error // Argument of type {onChange: () => void;} is not assignable to parameter of type Props | Readonly<Props>
+        const widget = new ExplanationWidgetExports.widget({
+            onChange: () => {},
         });
-        await userEvent.click(collapseLink); // collapse
+        const callbackMock = jest.fn();
+        const changeMock = jest
+            .spyOn(Changeable, "change")
+            .mockImplementation(() => {});
+
+        // Act - call the widget's "change" function
+        widget.change("foo", "bar", callbackMock);
+
+        // Assert
+        expect(changeMock.mock.contexts[0]).toEqual(widget);
+        expect(changeMock).toHaveBeenCalledWith("foo", "bar", callbackMock);
+
+        // Clean-up
+        changeMock.mockRestore();
```

Comment:
Our Jest tests automatically do this, so this is unnecessary. 

```suggestion
```

Reply:
Removed.